core/internal/features_test.go

```diff
@@ -933,7 +933,7 @@ func TestIntegration_FluxMonitor_NewRound(t *testing.T) {
 	availableFunds := minPayment * 100
 
 	// Start, connect, and initialize node
-	sub.On("Err").Return(nil)
+	sub.On("Err").Return(nil).Maybe()
```

Comment:
Seen this flake very rarely. The other FM test has a maybe also so just making them consistent.